src/legacy/ui/public/vis/editors/default/sidebar.js

```diff
@@ -76,6 +77,21 @@ uiModules.get('app/visualize').directive('visEditorSidebar', function () {
         }
 
         aggs.splice(index, 1);
+
+        if (agg.schema.group === AggGroupNames.Metrics) {
+          const metrics = $scope.state.aggs.bySchemaGroup(AggGroupNames.Metrics);
+          const enabledAggsCount = _.reduce(
+            metrics,
+            (count, aggregation) => {
+              return aggregation.enabled ? ++count : count;
+            },
+            0
+          );
+
+          if (!enabledAggsCount && metrics.length) {
```

Comment:
Would it be clear to have `if enabledAggsCount > 0`?

Reply:
This condition is checked whether there are no enabled aggs, e.g. `enabledAggsCount === 0`. If it's true, we make the first one is enabled.